src/main/java/com/vinylteam/vinyl/util/ParserConfiguration.java

```diff
@@ -0,0 +1,19 @@
+package com.vinylteam.vinyl.util;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.Value;
+
+@Value
+@Builder
+public class ParserConfiguration {
+    String vinylGenresSelector;
+    String offerLinkSelector;
+    String releaseSelector;
+    String artistSelector;
+    String catalogNumberSelector;
+    String priceDetailsSelector;
+    String highResolutionImageSelector;
+    String inStockMarkerSelector;
+    String inStockMarker;
```

Comment:
Please add empty row after class declaration and before closing, according to common codestyle.